### ISSUE TITLE
Strange colors on real Asus Zenphone2

### ISSUE THREAD

**Issue by michelmoorlag** [Brainflab/XamaRed.Forms.Svg#3]
On some Android simulators there is an issue showing the correct colors of svg. But on a real Asus Zenphone 2 I have found the same issue. See the screenshot. The icons should be black but are shown in a lila color. This is a major issue since this is a real device and not a simulator. I can not release my app if I can not be sure this works on all real devices.
![zenphone1](https://cloud.githubusercontent.com/assets/14913939/25942257/a7eec372-363c-11e7-900e-118ee607c388.png)
![zenphone2](https://cloud.githubusercontent.com/assets/14913939/25942262/ac5895f0-363c-11e7-9819-e136909d1bde.png)

**Comment by michelmoorlag** [comment]
It is related to Skiashap version 1.57.1. I am using an other plugin to show a svg. With skiasharp 1.57.1 I have exactly the same issue. When I downgrade to version 1.56.1 it all works fine! So maybe you should downgrade to this version too?

**Comment by Brainflab** [comment]
Great finding, thank you.
I was about to create Xamarin UI tests to check other devices.
I'll downgrade the dependency requirements to 1.56.1 later today.
Since I can't see any related issue in SkiaSharp GitHub I'll also create one.

**Comment by michelmoorlag** [comment]
Yeah if you could release a new version that can work with SkiaSharp 1.56.1 that would be really great! Please let us know when it's ready.
And of course something was messed up with SkiaSharp 1.57.1 and they should know about it and  hopefully fix it.

**Comment by Brainflab** [comment]
I uploaded a v1.0.1 on NuGet which lowers SkiaSharp dependencies requirement. I can't use 1.56.1 myself as my SVGs aren't supported on this version. But I hope it fixes your color issues.

**Comment by michelmoorlag** [comment]
I can confirm that with Skiasharp 1.56.1 colors are fine. So this is definitely something related to SkiaSharp 1.57.1.
I added a new issue to SkiaSharp: https://github.com/mono/SkiaSharp/issues/300

**Comment by michelmoorlag** [comment]
It should be fixed in SkiaSharp 1.58.1. See https://github.com/mono/SkiaSharp/issues/284

**Comment by Brainflab** [comment]
Thank you, looking forward to it
